### DISCUSSION TITLE
Question: Retain size of the window where it is moved to

### DISCUSSION THREAD

**Discussion by nliechti** [nikitabobko/AeroSpace#638] [questions-and-answers]
Hey sorry for yet another question.
My brain works in some kind of working "areas" where I want the windows placed. Most of the time i want a window in the center for example, occasionally exchanging it with another one there.
Is there any to kind of "swap" two windows instead of moving it? I didn't find any way yet to achieve this with aerospace.

**Comment by hwamil** [comment]
Closest thing would be `focus-back-and-forth` but that does not move windows. You would have to manually move the windows to the layout you want.

edit: by "manually" I mean use Aerospace's move commands, but you can also use your mouse.
